on-target-tests/Cargo.toml

```diff
@@ -70,4 +71,6 @@ nostd_async = { version = "0.6.1", features = ["wfe"] }
 futures = { version = "0.3.30", default-features = false, features = [
   "async-await",
 ] }
+i2c-write-iter = "1.0.0"
 itertools = { version = "0.12.0", default-features = false }
+portable-atomic = { version = "1.6.0", features = ["critical-section"] }
```

Comment:
This seems to be unused. At least compiling the tests still works if I comment this line out.

Reply:
In fact, I can revert all changes to this file and it still builds.

Reply:
Good catch, thanks!

Reply:
Oh, well the portable-atomic dep is redundant with the feature already enabled on heapless.
And it builds without those changes because the dum-dum I am forgot to add tests for this.

Reply:
Added some tests to check it actually works.